components/explore-the-data-page/DataDownloadButton.js

```diff
@@ -41,10 +44,29 @@ class DataDownloadButton extends React.Component {
           download={fileName}
           target="_blank"
           rel="noopener noreferrer"
-          onClick={() => this.startDownload(fileName)}
+          onClick={() => this.setState({ dataLicenseModalOpen: true })}
         >
           Download (CSV)
         </A>
+        {dataLicenseModalOpen && (
+          <Modal
+            button={{ text: 'Accept & Download', clickFunction: () => this.startDownload(fileName) }}
+            onClose={() => this.setState({ dataLicenseModalOpen: false })}
+          >
+            <div>
+              If you use TJI’s data, you must give TJI credit and adhere to TJI’s{' '}
```

Comment:
@EvaRuthM: The "Users:" at the beginning of this copy seemed unnecessary, so I dumped it. What do you think?

Reply:
@nholden fine with me - thanks!